src/main/java/dev/openfeature/sdk/Value.java

```diff
@@ -20,7 +20,7 @@
 @ToString
 @EqualsAndHashCode
 @SuppressWarnings({"PMD.BeanMembersShouldSerialize", "checkstyle:MissingJavadocType", "checkstyle:NoFinalizer"})
-public class Value implements Cloneable {
+public class Value {
```

Comment:
this change is ok, isn't it breaking change ?

Reply:
@Kavindu-Dodan I think @liran2000 is right, I didn't notice it but this does seem to be breaking.

Reply:
I think we need to revert the changes in this file.

Reply:
Reverted this so we are not breaking :)